c-pallets/segment-book/src/lib.rs

```diff
@@ -698,7 +712,7 @@ pub mod pallet {
 		fn trigger_challenge(now: BlockNumberOf<T>) -> bool {
 			const START_FINAL_PERIOD: Permill = Permill::from_percent(80);
 
-			let time_point = Self::random_time_number(20220509);
+			let time_point = Self::random_time_number(20220509); // where is this number coming from?
```

Comment:
In fact, this number is the date of the birth of this code.
Excuse me, do I need to pay attention to the selection of random number seeds.